### PR TITLE
Add a GitHub issue template

### PR DESCRIPTION
**Pull request by jd**
That should make sure users report the best information they can!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Thanks for taking the time for reporting an issue!
+
+Before reporting an issue on dd-trace-py, please be sure to provide all
+necessary information.
```
Comment (Kyle-Verhoog) [inline .github/ISSUE_TEMPLATE.md]: Can we be more specific about what "all necessary information" is? For bugs we could probably create a list of things to provide (library versions - maybe `requirements.txt` or `Pipfile`, python version, etc).

Comment (jd) [reply .github/ISSUE_TEMPLATE.md]: Good point, I'll add a question about `pip freeze`

Comment (brettlangdon) [reply .github/ISSUE_TEMPLATE.md]: Well, below he asks specific questions.

I thought about asking for `requirements.txt`, but not sure it is entirely necessary always?

Comment (majorgreys) [reply .github/ISSUE_TEMPLATE.md]: How about we roll these into a `ddtrace-run doctor`?

Comment (brettlangdon) [reply .github/ISSUE_TEMPLATE.md]: well, when we have it, sure :)

**Comment by brettlangdon** [comment]
We can always tweak these questions later

**Pull request merged**